### DISCUSSION TITLE
Film with 2 languages

### DISCUSSION THREAD

**Discussion by Bezman1970** [smplayer-dev/smplayer#706] [General]
Hi there,

When you watch a film that has two languages one english and one italian is there a way to just get english subtitles for the italian parts of the film and not the english parts.

Thanks for any advice

**Comment by Raza-0007** [comment]
Yes, you may need to search and download them from internet for the specific title you are watching. They are usually labeled as "forced subtitles" or "foreign language only" subtitles. Then just load them into smplayer during playback.
